app/test_main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+from pytest import MonkeyPatch
+
+import app.main
+
+
+def test_successfully_access_google_page(monkeypatch: MonkeyPatch) -> None:
```

Comment:
Consider renaming the test function to `test_valid_url_and_connection_exists` for better clarity and to align with the checklist's recommendation for descriptive test names.